frontend/packages/dev-console/integration-tests/support/constants/add.ts

```diff
@@ -65,3 +66,10 @@ export enum builderImages {
   Java = 'Java',
   NodeJs = 'Node.js',
 }
+
+export enum eventSourceCards {
+  ApiServerSource = 'Api Server Source',
```

Comment:
```suggestion
  ApiServerSource = 'ApiServerSource',
```

Reply:
event source card name is displaying same as shown below
![image](https://user-images.githubusercontent.com/61005404/111655091-5c669800-882f-11eb-909c-bc14dc4728b4.png)